modules/gcp-gke/main.tf

```diff
@@ -204,10 +207,6 @@ data "google_container_cluster" "current_cluster" {
   location = google_container_cluster.primary.location
 }
 
-moved {
-  from = google_container_node_pool.primary_node_pool
-  to   = google_container_node_pool.primary_node_pool.0
-}
```

Comment:
@MXfive  This change is made 6 month back and state is already migrated , So I guess it won't make any breaking changes.

Reply:
Yes, we need to check and you have so that's fine for when we upgrade cloud infra, but it's still a breaking change and so we still should follow the pattern and make the version bump accordingly.

Reply:
I have added this